src/Symfony/Component/HttpFoundation/Session/Storage/Handler/MongoDbSessionHandler.php

```diff
@@ -171,7 +163,34 @@ public function write($sessionId, $data)
     /**
      * {@inheritdoc}
      */
-    public function read($sessionId)
+    public function updateTimestamp($sessionId, $data)
+    {
+        $expiry = $this->createDateTime(time() + (int) ini_get('session.gc_maxlifetime'));
```

Comment:
This is duplicated code from function `doWrite` (line 144). Would it not be better to put this common code in a new protected method? May be something like `createExpiryTime`.

Reply:
not worth it for one line IMHO